app/index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require_once(__DIR__ . '/src/Db.php');
+require_once(__DIR__ . '/src/XmlParser.php');
+require_once(__DIR__ . '/src/Command.php');
+
+$url = getenv('NEXTCLOUD_IDPS_METADATA_URL');
+
+if (empty($url)) {
+  echo 'idps metadata url is missing. Set NEXTCLOUD_IDPS_METADATA_URL environment variable to provide it.';
```

Comment:
I have no idea if this is working in a CLI context, but why not using the [`error_log` function](https://www.php.net/manual/fr/function.error-log.php) instead?

Reply:
+1  for logging : add a helper function that use error_log ?
(I miss a PSR-3 Logger here ^^)

Reply:
Yes having monolog should be much better here, but I do this I will then install `symfony/console` and then `pomm` and then something else and at the end using python should be a better idea for a simple script like that :laughing:

Reply:
Python is always a better idea for a script. :trollface:

Reply:
You are probably right but the first idea was to used `Nextcloud` API but I did an other way...

Reply:
Trolling apart, it looks like a relevant approach.

Reply:
@madmatah what the idea behind having a helper using `error_log` ?

Reply:
The helper idea was just to be able to switch between error_log / printf / echo / whatever in a single place.